en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md

```diff
@@ -0,0 +1,1060 @@
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>maxResults</td>
+            <td>Maximum number of results to include in the response.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>nextToken</td>
+            <td>Pagination token to request the next set of results.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallTimeout</td>
+            <td>The amount of time(in seconds) to allow the client to complete the execution of an API call.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <amazonsqs.listQueues configKey="CONNECTION_NAME">
+        <queueNamePrefix>{$ctx:queueNamePrefix}</queueNamePrefix>
+        <maxResults>{$ctx:maxResults}</maxResults>
+        <nextToken>{$ctx:nextToken}</nextToken>
+        <apiCallTimeout>{$ctx:apiCallTimeout}</apiCallTimeout>
+        <apiCallAttemptTimeout>{$ctx:apiCallAttemptTimeout}</apiCallAttemptTimeout>
+    </amazonsqs.listQueues>
+    ```
+    
+    **Sample request**
+
+    ```xml
+    <amazonsqs.listQueues configKey="AMAZON_SQS_CONNECTION"/>
+    ```
+---
+
+??? note "purgeQueue"
+    This operation deletes available messages in a queue specified by the QueueURL parameter. See the [related API documentation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) for more information.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>queueUrl</td>
+            <td>The URL of the queue from which the PurgeQueue action deletes messages.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>apiCallTimeout</td>
+            <td>The amount of time(in seconds) to allow the client to complete the execution of an API call.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <amazonsqs.purgeQueue configKey="CONNECTION_NAME">
+        <queueUrl>{$ctx:queueUrl}</queueUrl>
+        <apiCallTimeout>{$ctx:apiCallTimeout}</apiCallTimeout>
+        <apiCallAttemptTimeout>{$ctx:apiCallAttemptTimeout}</apiCallAttemptTimeout>
+    </amazonsqs.purgeQueue>
+    ```
+    
+    **Sample request**
+
+    ```xml
+    <amazonsqs.purgeQueue configKey="AMAZON_SQS_CONNECTION">
+        <queueUrl>https://sqs.us-west-2.amazonaws.com/882234390511/Queue</queueUrl>
+    </amazonsqs.purgeQueue>
+    ```
+---
```

Comment:
Shall we add the error codes as well? Please refer http://localhost:8000/en/latest/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-reference/#error-codes-related-to-microsoft-azure-storage-connector as an example

Reply:
Added.